docs/redacted/backend.md

```
@@ -0,0 +1,169 @@
+
+---
+
+### `GET /health`
+
+```json
+{ "status": "ok" }
+```
+
+## GCS Storage Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)